docs/document/docs/components/desktop/attachment.md

```diff
@@ -1,10 +1,77 @@
+
+### Property
+Here is a list of properties that can be used for modifying the component:
+
+| Name   | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| className | string | ""  | Component class name | |
+| error | string | ""  | Text to be displayed in error | Error will not be displayed if unspecified or left empty |
+| id | string | ""  | Component id name | |
+| label | string | ""  | Label for the component | Label will not be displayed if unspecified or left empty |
+| language | string | "auto"  | Language setting | The text of “Browse“ button and “Drag & drop zone” will be changed according to language property<br>Available options: "auto", "en", "ja", "zh", "zh-TW"<br>If setting "auto", it will be according to the HTML lang setting (If the lang setting is other than "en"/"zh"/"zh-TW"/"ja", the language setting will be "en") |
+| files | Array<File \| {name: string; size: string; [key:string]: any}> | [] | The files of the selected | [File objects](https://developer.mozilla.org/en-US/docs/Web/API/File)<br>Or objects contain "name" and "size"<br>ex:<br>const fileObject = new File(["foo"], "foo.txt", {type: "text/plain"});<br>const customFile = {name:"foo.txt", size: "150", type: ".jpg"}<br> const attachment = new Attachment({files:[fileObject, customFile]});
+| disabled | boolean | false | Enable/Disable the component | |
+| requiredIcon | boolean | false | Show/Hide the required icon | |
+| visible | boolean | true | Show/Hide the component | |
+
+### Event
+
+Here is a list of events that can be specified:
+
+| Name | Type | Description | Remark |
+| :--- | :--- | :--- | :--- |
+| change | function | Event handler when the selected files has been changed | It will pass the event object as the argument<br>You can receive the following values in event.detail <li>add-file(Triggered if select any file)  <ul><li>event.detail.type: add-file<li>event.detail.oldFiles : Files before added<li>event.detail.files : Files after added<li>event.detail.fileIndex: Index of the added files(Type: number[])<ul><li>We can get the added files by "event.detail.files[event.detail.fileIndex[0]]"</ul></li></ul><li>remove-file(Triggered if remove any file)<ul><li>event.detail.type: remove-file<li>event.detail.oldFiles : Files before removed<li>event.detail.files : Files after removed<li>event.detail.fileIndex: Index of the removed file(Type: number[])<ul><li>We can get the removed file by "event.detail.oldFiles[event.detail.fileIndex[0]]"</ul></li></ul></li> |
+
+### Constructor
+
+Attachment(options)<br>
+Here is a list of available constructors:
+
+#### Parameter
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| options  | object | {} | Object that includes component properties |  |
+
+---
+## Sample Code
+
+> Please check the [package installation](../../getting-started/quick-start.md#installation) method first.
+
+Here is a sample code when all parameters are specified:
+``` javascript
```

Comment:
Please refer to the document of other components to add "const Kuc = Kucs['1.x.x'];" here.